pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTransactions.java

```diff
@@ -41,8 +42,30 @@ void run() throws Exception {
         }
     }
 
+    @Parameters(commandDescription = "Get transaction in pending ack stats")
+    private class GetTransactionInPendingAckStats extends CliCommand {
+        @Parameter(names = {"-c", "--coordinator-id"}, description = "the coordinator id", required = true)
+        private int coordinatorId;
+
+        @Parameter(names = {"-id", "--sequencce-id"}, description = "the sequenceId", required = true)
+        private int sequenceId;
```

Comment:
`sequenceId` is confusing here because this is also a  concept in the producer side, it's better to use `--transactionId` here? Such as --transactionId (0,123456)